### ISSUE TITLE
Textures: Blue spots on Imperator feathers

### ISSUE THREAD

**Issue by Decollates** [Decollates/MOD-ARK-Nyctatyrannus#8]
- Imperator has spots/coloration on its feathers that seem to replicate the bioluminescent spots. These markings do not glow. (Screenshots provided)
![346110_20231112175846_1](https://github.com/Decollates/MOD-ARK-Nyctatyrannus/assets/44717221/ea1683ad-a731-479c-8c84-e6acf783504d)

**Comment by Decollates** [comment]
Need to see about possibly having GameVideosForLife adjust the textures for this.

**Comment by bluescapes** [comment]
Yeah, some feather touchups would definitely be nice. The textures themselves dont quite work with the feathers, and I have to sub in stuff like the atlas to make it transparent and whatnot.

**Comment by Decollates** [comment]
> Yeah, some feather touchups would definitely be nice. The textures themselves dont quite work with the feathers, and I have to sub in stuff like the atlas to make it transparent and whatnot.

Would it be possible to remake them entirely with the same feathers in vanilla Ark, like on the Argent, Griffin, or Snow Owl?
